engine.py

```diff
@@ -23,200 +19,240 @@
 import nukescripts
 
 class NukeEngine(tank.platform.Engine):
+    """
+    An engine that supports Nuke 6.3v5+ and Hiero 9.0v*+.
+    """
+    # Define the different areas where menu events can occur in Hiero.
+    (HIERO_BIN_AREA, HIERO_SPREADSHEET_AREA, HIERO_TIMELINE_AREA) = range(3)
 
-    ##########################################################################################
-    # init
+    def __init__(self, *args, **kwargs):
+        self._hiero = nuke.env.get("hiero")
```

Comment:
is this a boolean? In that case I think it should be saying `self._hiero_enabled` or something like that. `self._hiero` sounds to me like it contains the hiero "object".